backoffice/src/controllers/auth/index.js

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+
+/**
+ * Call yakchat auth server
+ */
+export function login (username, password) {
+	axios.post('http://localhost:8080/api/api/v1/auth/login', 
```

Comment:
The URL should be under a config object ... on build process the app should use production URL and  point to localhost only on development

Reply:
is this resolved?

Reply:
yes, is the same bellow, added to the env file